### PR TITLE
Doc: Fix: Persist the state of theme even after refresh

### PR DESCRIPTION
**Pull request by sivareddyuppathi**
Fix: Persist the state of theme even after refresh

Fix for bug number (https://github.com/primefaces/primevue/issues/6808)

### CODE REVIEW

File: apps/showcase/app.vue [hunk 1]
```diff
@@ -25,13 +29,17 @@ export default {
             document.startViewTransition(() => this.toggleDarkMode(event));
         },
         toggleDarkMode(event) {
-            const isDark = event.dark;
+            const isDark = event?.dark;
+
+            console.log(event, 'aaaa');
```
Comment (j0rgedev) [inline apps/showcase/app.vue:34]: Don't forget to remove this log

**Comment by j0rgedev** [comment]
I hope this get merged. Last time, they rejected it, but I don't remember why and can't find the PR

File: apps/showcase/app.vue (hunk 1 above)
Comment (sivareddyuppathi) [reply apps/showcase/app.vue]: Fone @j0rgedev

**Comment by tugcekucukoglu** [comment]
We do not want to keep it in local storage.

Thank you for your contributions.

**Comment by chocolateimage** [comment]
> We do not want to keep it in local storage.
> 
> Thank you for your contributions.

Why not in localStorage? Google, Vue, React, etc.. does it in localStorage

**Pull request closed without merging**